### DISCUSSION TITLE
Метательное оружие для граждан

### DISCUSSION THREAD

**Discussion by Starikprostoy** [octothorp-team/public-dobrograd#639] [Предложения]
Я предлагаю добавить такие вещи как молотов, кирпич, дымовую шашку. Как по мне, это повысит виды для РП и например поджоги не будут состоять из "/me облил здание канистрой, кинул окурок", а уже из "/me достал коктейль молотова, поджег, закинул в здание" и это уже будет наносить дамаг по бизнесу и заставит РПшить, а не так просто забивать на отыгровки и прочее, появится работа для DFD и прочего. С кирпичем же можно сделать так-это метательное оружие, наносящее 1/5 ХП при попадании в тело, 1/8 ХП при попадании в конечность и временное ее обезвреживание(как с стрельбой, попало в руку-выбивает оружие, попало в ногу-замедляет), и 1/3-фулл ХП в голову, кирпичи можно будет использовать во время бунтов и митингов. Сразу скажу, кирпич не должен лететь прямо в голову, ибо это будет новым оружием для обходчиков. Дымовые шашки-аналог слезоточивого газа для СВАТа. Все это будет либо крафтится у токаря, либо покупаться, а может и то и другое. Для их использования можно сделать ограничение по времени, например 30 часов для шашки, 25 часов для кирпича и 40-50 часов для молотова(Я уверен что обходчики не будут играть день дабы просто покидать кирпичи)

**Comment by Kyper1** [comment]
Неплохо, но вот время, по-моему, слишком мало, постоянные поджоги и.т.д. будут слишком нагружать сервер, потому что глеки отмычкины, наигравшие по 30 часов, будут то и делать, что вместо грабежей в переулках заниматься поджогами. Я бы повысил время до 150ч игры на сервере, чтобы получилось взять руки все, перечисленное тобою.

**Reply by Starikprostoy** [reply]
Чтобы кинуть кирпич, нужно прожить в городе пол года IC... Круто) С молотовым я согласен, слишком опасное, но 150 часов это слишком. Дымовая шашка пуст будет 50-70 часов, кирпич 40-50 часов, молотов 90-100 часов, как по мне это более менее приемлимо

**Reply by Kyper1** [reply]
Даже если так, то дымовую шашку нужно поменять с кирпичом местами

**Reply by Starikprostoy** [reply]
Вообщем, если Челог одобрит, он сам будет решать сколько времени надо

**Comment by LENFYS** [comment]
За

**Comment by travka4000** [comment]
Я за, эсли от примера коктеля молотова то получиться пожарные нужны будут, ведь пожары делают контент менеджеры и они берут пожарных
